add-ons/inventory/src/mender-inventory.c

```diff
@@ -148,6 +174,37 @@ mender_inventory_set(mender_keystore_t *inventory) {
     return ret;
 }
 
+mender_err_t
+mender_inventory_set_item(char *name, char *value) {
+
+    mender_err_t ret;
+
+    /* Take mutex used to protect access to the inventory key-store */
+    if (MENDER_OK != (ret = mender_scheduler_mutex_take(mender_inventory_mutex, -1))) {
+        mender_log_error("Unable to take mutex");
+        return ret;
+    }
+
+    int index;
```

Comment:
It's better to have the declaration at the beginning of the function.

Reply:
Fixed